### PR TITLE
Dunking on react

### PR DESCRIPTION
**Pull request by wpdunk**
"if the code already exists don't rewrite it" - agile docs wizard

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -22,5 +28,13 @@
     "not dead",
     "not ie <= 11",
     "not op_mini all"
-  ]
+  ],
+  "devDependencies": {
+    "cypress": "^3.2.0"
```
Comment (punchcafe) [inline package.json]: shouldn't this be dead?

File: src/App.js [hunk 2]
```diff
@@ -1,23 +1,14 @@
...
-          <a
-            className="App-link"
-            href="https://reactjs.org"
-            target="_blank"
-            rel="noopener noreferrer"
-          >
-            Learn React
-          </a>
-        </header>
+        Here's the webcam
```
Comment (jesslns) [inline src/App.js]: Do we need a <WebcamFeed> here?

Comment (ellieturno) [reply src/App.js]: React components only have one <> component tag. Only need the one <WebcamFeed /> here.

Comment (wpdunk) [reply src/App.js]: nope

**Pull request closed without merging**